src/transactions/saga.ts

```diff
@@ -256,7 +256,7 @@ export function* getTransactionReceipt(
   const { feeCurrencyId, transactionHash } = transaction
 
   try {
-    const receipt = yield* call([publicClient[network], 'getTransactionReceipt'], {
+    const receipt = yield* call([publicClient[network], 'waitForTransactionReceipt'], {
```

Comment:
this is kind of unrelated but since we are polling for the tx receipt, it's expected that at the beginning the receipt is not there. this change prevents the catch block from being hit unnecessarily